js/index.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+
+	var cityAustin = true if 
+	var cityLA = true
+	var citySF = true
+	var citySydney = true
+
+	$('#city-type').submit(addEntry)
```

Comment:
`submit` handling only applies to `<form>` tags. This handler is being attached to the `<input>` tag that has the id `city-type`.